samples/Chat/package.json

```diff
@@ -15,6 +15,7 @@
     "test:ci-coverage": "echo skip",
     "preprocess": "cpx \"./src/**\" ./preprocessed && babel ./src --out-dir ../preprocessed --extensions \".ts,.tsx\" --keep-file-extension --config-file ./.babelrc.js --relative",
     "preprocess:stable-flavor": "rushx _stable-flavor \"rushx preprocess && rushx lint:fix && rushx prettier\"",
+    "port:sample": "rushx preprocess:stable-flavor && cpx \"./preprocessed/**\" sample-dist/src && cpx \"./Media/**\" sample-dist/Media && cpx \"./public/**\" sample-dist/public",
```

Comment:
Not related to this change, but when I am reading _stable-flavor file it is like:

"_by-flavor": "rushx _current-flavor && env-cmd -f ../../common/config/env/.env --use-shell",

It shows you current flavor then run the command under stable flavor - which might be confusing, I think we can remove "rushx _current-flavor &&" part of it, but doesn't have to be in this PR

Reply:
@PorterNan  can we log this as a bug that we can close quite quickly? I dont want to lose track of this suggestion.